README.md

```diff
@@ -7,35 +7,68 @@ so that you can extract an error catalog from the code.
 
 ## In a Nutshell
 
-Create an error object:
+### Install the library
 
-```python
-exa_error_obj = ExaError.message_builder('E-TEST-1')
-    .message("Not enough space on device {{device}}.")
-    .mitigation("Delete something from {{device}}.")
-    .mitigation("Create larger partition.")
-    .parameter("device", "/dev/sda1", "name of the device") 
+```shell
+pip install exasol-error-reporting
 ```
 
-Use it as string:
+### Create a Simple Error
+
+```python
+from exasol import error
+
+error1 = error.ExaError(
+    "E-TEST-1", "A trivial error", "No mitigation available", {}
+)
```

Comment:
Is it possible to omit the empty dict `{}` and the mitigation? If yes: remove both and add separate examples with mitigation and params.

Reply:
currently it is not supported, due to the fact that this would require additional code when analyzing the ast.
We can create an issue for it though, just wanted to wait if this will be requested by somebody else than the implementer ;).